[1.8.0] Error: Cannot find module 'eslint-webpack-plugin'

With the latest 1.8.0 release we're getting following error:

```
[webpack-cli] Failed to load '\test\webpack.config.js' config
[webpack-cli] Error: Cannot find module 'eslint-webpack-plugin'
```

With `"@symfony/webpack-encore": "1.7.1",` it works out of the box.

It looks like when the dependency `"eslint-webpack-plugin": "^3.1.1"` in `package.json` is missing it fails, eventho the `WebpackConfig.useEslintLoader` and `WebpackConfig.useEslintPlugin` is set to `false` by default and never touched.

I created locally a quick test case and stripped everything (sass, react, ...) out of it to ensure that it's failing with the bare minimum

----

**package.json**

```json
{
    "name": "test",
    "license": "GPL-2.0-or-later",
    "author": {
        "name": "Christian Leucht",
        "email": "c.leucht@inpsyde.com",
        "role": "Developer"
    },
    "devDependencies": {
        "@symfony/webpack-encore": "^1.8.0",
        "ts-loader": "^9.0.0",
        "typescript": "^3.3.4000"
    }
}
```

-----

**webpack.config.js**

```js
const Encore = require('@symfony/webpack-encore');

Encore
    .setOutputPath('assets/')
    .setPublicPath('/assets/')
    .addEntry('test', './resources/ts/test.ts')
    .enableTypeScriptLoader()
    .enableSingleRuntimeChunk()
;

module.exports = Encore.getWebpackConfig();
```

---

**tsconfig.json**

[!] Not executed, but in case you want to change it to `"@symfony/webpack-encore": "1.7.1",` for testing:

```json
{
    "compilerOptions": {
        "target": "es5",
        "module": "commonjs",
        "lib": [
            "es5",
            "es6",
            "dom"
        ],
        "moduleResolution": "node",
        "declaration": false,
        "esModuleInterop": true,
        "noImplicitAny": false,
        "sourceMap": true,
        "noLib": false,
        "suppressImplicitAnyIndexErrors": true
    },
    "compileOnSave": false,
    "exclude": [
        "node_modules",
        "vendor"
    ]
}
```

----

## Additional infos

```
>yarn list --pattern webpack-encore 
yarn list v1.22.15
└─ @symfony/webpack-encore@1.8.0
```

----

## Stack

Here's the error stack (cleaned up the path):

```cli
Require stack:
- \test\node_modules\@symfony\webpack-encore\lib\plugins\eslint.js
- \test\node_modules\@symfony\webpack-encore\lib\config-generator.js
- \test\node_modules\@symfony\webpack-encore\index.js
- \test\webpack.config.js
- \test\node_modules\webpack-cli\lib\webpack-cli.js
- \test\node_modules\webpack-cli\lib\bootstrap.js
- \test\node_modules\webpack-cli\bin\cli.js
- \test\node_modules\webpack\bin\webpack.js
- \test\node_modules\@symfony\webpack-encore\bin\encore.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (\test\node_modules\@symfony\webpack-encore\lib\plugins\eslint.js:15:22)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19) {
    '\test\\node_modules\\webpack-cli\\lib\\webpack-cli.js',
    '\test\\node_modules\\webpack-cli\\lib\\bootstrap.js',
    '\test\\node_modules\\webpack-cli\\bin\\cli.js',
    '\test\\node_modules\\webpack\\bin\\webpack.js',
    '\test\\node_modules\\@symfony\\webpack-encore\\bin\\encore.js'
  ]
}

```

--- a/lib/plugins/eslint.js
+++ b/lib/plugins/eslint.js
@@ -12,7 +12,6 @@
 const forceSync = require('sync-rpc');
 const hasEslintConfiguration = forceSync(require.resolve('../utils/has-eslint-configuration'));
 const WebpackConfig = require('../WebpackConfig'); //eslint-disable-line no-unused-vars
-const EslintPlugin = require('eslint-webpack-plugin'); //eslint-disable-line node/no-unpublished-require
 const applyOptionsCallback = require('../utils/apply-options-callback');
 const pluginFeatures = require('../features');
 
@@ -54,6 +53,8 @@ Install ${chalk.yellow('babel-eslint')} to prevent potential parsing issues: ${p
             emitWarning: true,
             extensions: ['js', 'jsx'],
         };
+
+        const EslintPlugin = require('eslint-webpack-plugin'); //eslint-disable-line node/no-unpublished-require
 
         plugins.push({
             plugin: new EslintPlugin(

